lib/Doctrine/Common/Persistence/Mapping/AbstractClassMetadataFactory.php

```diff
@@ -398,16 +395,28 @@ public function getReflectionService()
     }
 
     /**
-     * Gets the real class name of a class name that could be a proxy.
+     * Gets the real class name of a class name that could be a proxy or alias.
      */
-    private function getRealClass(string $class) : string
+    protected function getRealClassName(string $className) : string
```

Comment:
This should most likely stay `private`

Reply:
This is protected as per suggestion of @alcaeus at https://github.com/doctrine/persistence/pull/21#discussion_r249341149